src/mongosh/schemas/curriculum-2.0.0.json

```diff
@@ -1,33 +1,49 @@
 {
-	"description": "A Learning resource or other activity that is on a persons career or learning roadmap",
+	"description": "A Learning resource or other activity that is on a person's career or learning roadmap",
 	"bsonType": "object",
 	"required": [
-		"_id"
+		"_id",
+		"personId",
+		"resources"
 	],
 	"properties": {
 		"_id": {
 			"description": "The unique identifier for a resource used by a person",
 			"bsonType": "objectId"
 		},
-        "version": {
-			"description": "Scehma Version (only on Version document)",
-            "bsonType": "string",
-            "maxLength": 20
-        },
+		"version": {
+			"description": "Schema Version (only on Version document)",
+			"bsonType": "string",
+			"maxLength": 20
+		},
 		"personId": {
-			"description": "The ID of the person this item was asigned to",
+			"description": "The ID of the person this item was assigned to",
 			"bsonType": "objectId"
 		},
-		"resourceId": {
-			"description": "The ID of the resource asigned",
-			"bsonType": "objectId"
+		"resources": {
+			"description": "An array of resources",
+			"bsonType": "array",
+			"items": {
+				"bsonType": "object",
+				"properties": {
+					"resourceId": {
+						"description": "The ID of the resource assigned",
+						"bsonType": "objectId"
+					},
+					"topicId": {
+						"description": "The ID of the topic associated with the resource",
+						"bsonType": "objectId"
+					}
+				},
+				"required": ["resourceId", "topicId"]
+			}
```

Comment:
A few things here:
- Be cautious about required properties, keep in mind auto-save features
- I think the resource object in the array should have a ``type`` attribute that is ``topic`` or ``adhoc`` 
- for "topic" type resources we have a resourceId, topicId
- for "adhoc" type resources we have a name, link
- all resource objects should have a review and roadmap property
- roadmap values should be enumerated to 
   - completed
   - now
   - next
   - later

this is a more complex structure and will require some additional knowledge. Research json schema and the ``oneOf`` construct.